### PR TITLE
chore: add route http interface & fix some bugs

### PR DESCRIPTION
**Pull request by ZuLiangWang**
# Which issue does this PR close?

Closes #

# Rationale for this change
Fix some bugs introduced due to refactoring code, enables the cluster to be deployed normally.
# What changes are included in this PR?
* The values in type enum and pb do not correspond., correct the enum values in type, make them consistent with pb.
* Add http route interface.

# Are there any user-facing changes?
None.

# How does this change test
Local manual test.

### CODE REVIEW

File: server/service/http/api.go [hunk 1]
```diff
@@ -144,3 +146,35 @@ func (a *API) transferLeader(writer http.ResponseWriter, req *http.Request) {
 
 	a.respond(writer, "ok")
 }
+
+type RouteRequest struct {
+	ClusterName string `json:"clusterName"`
+	SchemaName  string `json:"schemaName"`
+	Table       string `json:"table"`
```
Comment (chunshao90) [inline server/service/http/api.go:153]:
```suggestion
	Tables       []string `json:"tables"`
```

File: server/service/http/api.go [hunk 2]
```diff
@@ -144,3 +146,35 @@ func (a *API) transferLeader(writer http.ResponseWriter, req *http.Request) {
...
+}
+
+func (a *API) route(writer http.ResponseWriter, req *http.Request) {
+	var routeRequest RouteRequest
+	err := json.NewDecoder(req.Body).Decode(&routeRequest)
+	if err != nil {
+		log.Error("decode request body failed", zap.Error(err))
```
Comment (chunshao90) [inline server/service/http/api.go:160]:
```suggestion
		log.Error("decode request body failed", zap.Error(err),zap.String("body", string(req.Body)))
```

File: server/service/http/api.go [hunk 3]
```diff
@@ -144,3 +146,35 @@ func (a *API) transferLeader(writer http.ResponseWriter, req *http.Request) {
...
+	if err != nil {
+		log.Error("route tables failed", zap.Error(err))
+		a.respondError(writer, cluster.ErrRouteTable, nil)
+		return
+	}
+
+	resultByte, err := json.Marshal(result)
+	if err != nil {
+		log.Error("route tables result marshal failed", zap.Error(err))
+		a.respondError(writer, ErrParseResponse, nil)
```
Comment (chunshao90) [inline server/service/http/api.go:175]: Add some extra info.

File: server/service/http/api.go (hunk 2 above)
Comment (ZuLiangWang) [reply server/service/http/api.go:160]: Request body has been print in `printRequestInsmt`, so it is not need here.

**Review by chunshao90 — APPROVED** [review]
LGTM

**Pull request merged**